prod/westeurope/internal/api/functions_admin/function_app/terragrunt.hcl

```diff
@@ -114,6 +114,7 @@ inputs = {
     USER_DATA_BACKUP_CONTAINER_NAME = dependency.storage_container_user-data-backup.outputs.name
     USER_DATA_DELETE_DELAY_DAYS     = 7
 
+    SLOT_TASK_HUBNAME = "FunctionsAdminHub"
```

Comment:
sorry there was a misunderstanding :) I'd prefer to change the names here 
(ie. ProductionTaskHub) if we're gonna create a NEW function app service with the runtime upgraded to v3 (what do you think about it @francescopersico ?)

Reply:
@gunzip changed in ProductionTaskHub